qiskit_ibm/runtime/program/program_metadata_sample.json

```diff
@@ -3,15 +3,46 @@
   "description": "Simple runtime program used for testing.",
   "max_execution_time": 300,
   "version": 1.0,
-  "backend_requirements": {"min_num_qubits":  5},
-  "parameters": [
-    {"name": "iterations", "description": "Number of iterations to run. Each iteration generates a runs a random circuit.", "type": "integer", "required": true}
-  ],
-  "return_values": [
-    {"name": "-", "description": "A string that says 'All done!'.", "type": "string"}
-  ],
-  "interim_results": [
-    {"name": "iteration", "description": "Iteration number.", "type": "int"},
-    {"name": "counts", "description": "Histogram data of the circuit result.", "type": "dict"}
-  ]
+  "spec": {
+    "backend_requirements": {
+      "min_num_qubits": 5
+    },
+    "parameters": {
+      "type": "object",
+      "properties": {
+        "iterations": {
+          "description": "Number of iterations to run. Each iteration generates a runs a random circuit.",
+          "type": "integer"
+        }
+      },
+      "required": [
+        "iterations"
+      ]
+    },
+    "return_values": {
+      "type": "object",
+      "properties": {
+        "-": {
+          "description": "A string that says 'All done!'.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "-"
+      ]
```

Comment:
So the return value of this program is not really something like

```
{"-": "All done!"}
```
It is just a string `All Done!`. 

Of course this is just a sample metadata, so it doesn't need to match the program this is based on. Since that's the case, using something that's not `-` is probably more helpful to the users.